pyproject.toml

```
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "autopush-notification-integration-tests"
+version = "0.1.0"
+description = "Autopush Notification Tests"
+authors = ["Benjamin Forehand Jr <bforehand@mozilla.com>"]
+license = "Mozilla Public License 2.0"
+readme = "README.md"
+package-mode = false
+
+[tool.poetry.dependencies]
+python = "^3.12"
+imgcompare = "^2.0.1"
+pyscreenshot = "^3.1"
+pytest = "^8.3.3"
+PyVirtualDisplay = "^3.0"
+pillow = "^11.0.0"
+pytest-selenium = "^4.1.0"
+
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
```

nit pin the latest stable poetry-core version
```suggestion
[build-system]
requires = ["poetry-core>=1.8.1"]
build-backend = "poetry.core.masonry.api"
```